### PR TITLE
Fix to import order

### PR DESCRIPTION
**Pull request by cgruber**

### CODE REVIEW

**Comment by cgruber** [comment]
Minor stuff in google's internal repo, likely altered by IDE settings automatically updating things.  Pushing out so we don't have superfluous merge conflicts.

File: common/src/test/java/com/google/auto/common/SuperficialValidationTest.java [hunk 1]
```diff
@@ -274,7 +274,7 @@ public void invalidAnnotationValue() {
   }
   */
 
-  private abstract static class AssertingProcessor extends AbstractProcessor {
```
Comment (eamonnmcmanus) [inline common/src/test/java/com/google/auto/common/SuperficialValidationTest.java]: Google style and the [JLS](http://docs.oracle.com/javase/specs/jls/se8/html/jls-8.html#jls-8.1.1) both say that this is already the correct order. Where is the other one coming from?

**Comment by cgruber** [comment]
FYI, @eamonnmcmanus - these are two no-op changes.  I'm going to merge them pre-LGTM (TBR essentially) so I can get a MOE equivalence.

**Comment by cgruber** [comment]
Oh, NVM.  You're doing your thing.

File: common/src/test/java/com/google/auto/common/SuperficialValidationTest.java (hunk 1 above)
Comment (cgruber) [inline common/src/test/java/com/google/auto/common/SuperficialValidationTest.java]: Um... dunno - but it got flipped internally.  Maybe we need to un-flip it.  :/  I'll revert this one change, and we can push it the other direction then.

**Comment by cgruber** [comment]
Ok - it's only the import order now.

**Pull request merged**